README.md

```diff
@@ -128,30 +176,30 @@ To contribute:
 
 (*) A good commit message is as follows:
 
-```
+```example
 Module: add/fix/remove a from b
 
 Explanation on what/how/why
 ```
 
 For example:
 
-```
+```example
 Video chip: implement 320x240 text-mode
 
 It is now possible to switch to 320x240 text-mode and display text.
 ```
 
-# License
+## License
 
-`Z80.js` file is distributed under the MIT license (originated from Molly Howell repository: https://github.com/DrGoldfire/Z80.js)
+`Z80.js` file is distributed under the MIT licence (originated from Molly Howell repository: [Z80.js](https://github.com/DrGoldfire/Z80.js))
 
 All the other files are distributed under the Apache 2.0 License. See `LICENSE` file for more information.
 
 You are free to use it for personal and commercial use, the boilerplate present in each file must not be removed.
 
-# Contact
+## Contact
 
-For any suggestion or request, you can contact me at contact [at] zeal8bit [dot] com
+For any suggestion or request, you can contact me at `contact@zeal8bit.com`
```

Comment:
This was done on purpose to avoid robot finding the email with a simple search

Reply:
Fixed.